web/src/components/areagraph.js

```diff
@@ -126,14 +157,7 @@ AreaGraph.prototype.data = function (arg) {
     }
     this.y.domain([
         0,
-        1.1 * d3.max(arg, function(d) {
-            if (!that._displayByEmissions) {
-                return d.totalProduction + d.totalImport + d.totalDischarge;
-            } else {
-                // in tCO2eq/min
-                return (d.totalCo2Production + d.totalCo2Import + d.totalCo2Discharge) / 1e6 / 60.0;
-            }
-        })
+        maxTotalValue
```

Comment:
can you multiply here by 1.1? It gives a bit of space between the graph and the axis (and then we keep the existing behavior)